test/test_lexer.ml

```diff
@@ -18,7 +18,7 @@ let next_token_tests =
   "next_token tests"
   >::: [
          make_next_token_test "empty program" "" [ EOF ];
-         make_next_token_test "program with whitespace" "   \n\t\r\n" [ EOF ];
+         make_next_token_test "program with whitespace" "   \n\r\t\n" [ EOF ];
```

Comment:
accidentally switched the original input but still tests same thing

Reply:
<img
  width="250px"
  src="https://media1.tenor.com/images/ddd439116a7e12625de54132e8a40026/tenor.gif?itemid=11156171"
/>

ACKCHTUALLYY, `\r\n` is the standard line ending sequence for text files on Windows. `\n\r` is not real